docker/Dockerfile

```diff
@@ -106,6 +109,14 @@ WORKDIR $CONTAINER_SOURCE/
 # Upstream only
 COPY --from=cleanup --chown=1001:1001 $CONTAINER_SOURCE/ ./
 
+# Copy python script used to gather dynamic plugins
```

Comment:
please remove the comment on line 96 above since "install techdocs dependencies" is no longer an upstream only thing (it's done differently downstream, but it WILL be downstream -- so we can have a different section entirely).

Reply:
Done.